### PR TITLE
Ollie dataset

### PR DESCRIPTION
**Pull request by ontocord**
This is the dataset used to train the Ollie open information extraction algorithm. It has over 21M sentences. See http://knowitall.github.io/ollie/ for more details.

### CODE REVIEW

File: datasets/ollie/README.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+https://knowitall.github.io/ollie/
+- **Repository:**
+https://knowitall.github.io/ollie/
+- **Paper:**
+@inproceedings{ollie-emnlp12,
+  author = {Mausam and Michael Schmitz and Robert Bart and Stephen Soderland and Oren Etzioni},
+  title = {Open Language Learning for Information Extraction},
+  booktitle = {Proceedings of Conference on Empirical Methods in Natural Language Processing and Computational Natural Language Learning (EMNLP-CONLL)},
+  year = {2012}
+}
```
Comment (lhoestq) [inline datasets/ollie/README.md:62]:
```suggestion
- **Homepage:** [Ollie](https://knowitall.github.io/ollie/)
- **Repository:** [Github](https://github.com/knowitall/ollie)
- **Paper:** [Aclweb](https://www.aclweb.org/anthology/D12-1048/)
```
Here we expect an url for the paper.
I also changed the repository url.

File: datasets/ollie/README.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+- n>1M
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+- text-retrieval
+task_ids:
+- entity-linking-retrieval
+- fact-checking-retrieval
+- other-stuctured-to-text
```
Comment (lhoestq) [inline datasets/ollie/README.md:22]: If I understand correctly the task is relation extraction right ?

```suggestion
task_categories:
- other
task_ids:
- other-other-relation-extraction
```

**Review by lhoestq — COMMENTED** [review]
Awesome thank you !

Looks all good to me :) 

I just left two suggestions for the dataset card, please let me know what you think and we can merge this one

File: datasets/ollie/README.md (hunk 2 above)
Comment (ontocord) [reply datasets/ollie/README.md:22]: Yes, I'll make the addition  but it's bidrectional though - it's relation extraction and structure to text.  For my purposes, I would also like to experiment with speeding up pre-training using extraction patterns.

File: datasets/ollie/README.md (hunk 1 above)
Comment (ontocord) [reply datasets/ollie/README.md:62]: Cool. I'll make the change.

**Review by lhoestq — APPROVED** [review]
Thanks :)

**Pull request merged**